.github/workflows/daily.yml

```
@@ -0,0 +1,25 @@
+# This workflow will run integrations tests daily at midnight.
+
+name: Daily CI
+
+on:
+  # JUST FOR TESTING THE WORKFLOW
+  pull_request:
+    branches: [master]
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  test:
+    name: Daily Server Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        server: [eu-ger-1, eu-ger-2, eu-fin-1, us-or-1, us-or-2, us-pa-1, us-pa-2, us-va-1]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Daily CI
+        uses: actions/setup-node@v1
+      - run: yarn
+      - run: SKYNET_JS_INTEGRATION_TEST_SERVER="https://${{ matrix.server }}.siasky.net" yarn run jest src/integration.test.ts
```

It looks cleaner when you put dependencies installation in its separate step and use `env` for environment variables.

```suggestion
      - uses: actions/checkout@v2
      - name: Use Node.js
        uses: actions/setup-node@v1

      - name: Install dependencies
        run: yarn
        
      - name: Daily CI
        run: yarn run jest src/integration.test.ts
        env:
          SKYNET_JS_INTEGRATION_TEST_SERVER: "https://${{ matrix.server }}.siasky.net"
```